jsapp/js/account/usage.api.ts

```diff
@@ -1,5 +1,6 @@
 import {fetchGet, fetchPost} from 'jsapp/js/api';
 import {getOrganization} from 'js/account/stripe.api';
+import {createContext} from 'react';
 
```

Comment:
Similarly to `stripe.api.ts` - some short `/**` comments for funfctions would be great.